src/app/(nextjs_migration)/(guest)/breach-details/[breachName]/page.tsx

```diff
@@ -0,0 +1,368 @@
+  "email-addresses": Glyph12,
+  "dates-of-birth": Glyph13,
+  pins: Glyph14,
+  "physical-addresses": Glyph15,
+};
+
+export default async function BreachDetail(props: {
+  params: { breachName: string };
+}) {
+  const l10n = getL10n();
+  const breachName = props.params.breachName;
+  const allBreaches = await getBreaches();
+  const breach = getBreachByName(allBreaches, breachName);
+  const breachLogos = await getBreachIcons(allBreaches);
+
+  return (
+    <div data-partial="breachDetail">
+      <header className="breach-detail-header">
+        <div className="breach-detail-meta">
+          <h1>
+            <BreachLogo breach={breach} logos={breachLogos} />
+            {breach.Title}
+          </h1>
+          {getBreachCategory(breach) === "website-breach" ? (
+            <a
+              href={`https://${breach.Domain}`}
+              className="breach-detail-meta-domain"
+              rel="nofollow noopener noreferrer"
+              data-event-label={breach.Domain}
+              data-event-action="Engage"
+              data-event-category="Breach Detail: Website URL Link"
+              target="_blank"
+            >
+              {breach.Domain}
+            </a>
+          ) : null}
+          <a
+            href="#what-is-this-breach"
+            className="breach-detail-meta-more-info"
+          >
+            {l10n.getString(getBreachCategory(breach))}
+          </a>
+        </div>
+      </header>
+
+      {/* Overview */}
+      <div className="breach-detail-overview">
+        <div className="breach-detail-overview-blurb">
+          <h2>{l10n.getString("breach-overview-title")}</h2>
+          <div>
+            {l10n.getString("breach-overview-new", {
+              breachDate: breach.BreachDate.toLocaleString(getLocale(), {
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              }),
+              breachTitle: breach.Title,
+              addedDate: breach.AddedDate.toLocaleString(getLocale(), {
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              }),
+            })}
+          </div>
+          {compareBreachDates(breach) ? (
+            <a href="#delayed-reporting">
+              {l10n.getString("delayed-reporting-headline")}
+            </a>
+          ) : null}
+        </div>
+
+        {/* Exposed Data Classes */}
+        <div>
+          <h3>{l10n.getString("what-data")}</h3>
+          <ul className="breach-detail-compromised-list">
+            {makeDataSection(breach)}
+          </ul>
+          <p
+            className="breach-detail-attribution"
+            dangerouslySetInnerHTML={{
+              __html: l10n.getString("email-2022-hibp-attribution", {
+                "hibp-link-attr":
+                  'href="https://haveibeenpwned.com/" target="_blank"',
+              }),
+            }}
+          />
+        </div>
+      </div>
+
+      {/* Sign Up Banner */}
```

Comment:
nit/random question: Do we want to use a different style of comments here? 
```suggestion tsx
{
  // Sign Up Banner
}
```

Reply:
I don't have a strong preference, just went with this style because that's what VSCode gives you if you press <kbd>Ctrl</kbd>+<kbd>/</kbd> :) Switched in 18f213af5